examples/sqlite.rs

```diff
@@ -50,7 +51,7 @@ impl SimpleQueryHandler for SqliteBackend {
     where
         C: ClientInfo + Unpin + Send + Sync,
     {
-        let conn = self.conn.lock().unwrap();
+        let conn = self.conn.lock().await;
```

Comment:
I think we don't have to switch to tokio async lock to solve this. 
Because the operation is fully memory based, it has no major difference between tokio and std.